templates/azure/sql.html

```diff
@@ -0,0 +1,202 @@
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">FIPS & CC-EAL2 certified</h3>
+          <div class="p-matrix__desc">
+            <p>Officially certified components to enable operating environments under compliance regimes like FedRAMP, HIPAA, PCI and ISO.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Screaming fast and optimised</h3>
+          <div class="p-matrix__desc">
+            <p>SQL Server on Ubuntu Pro is optimised for performance and stability, with out of the box support for PMEM-enabled instances where available.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Ready for the unexpected</h3>
+          <div class="p-matrix__desc">
+            <p>The solution includes a dedicated Azure high availability fencing agent, for better failover times and greatly improved reliability.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">CIS STIG</h3>
+          <div class="p-matrix__desc">
+            <p>For companies looking to leverage industry benchmarks for hardening, Ubuntu Pro makes two leading implementation guides  available.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">10-year lifetime</h3>
+          <div class="p-matrix__desc">
+            <p>Canonical backs SQL Server on Ubuntu Pro for up to 10 years, ensuring security updates are available throughout.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Retain governance</h3>
+          <div class="p-matrix__desc">
+            <p>You decide how and when patches, updates and other lifecycle actions take place - for full compliance and control.</p>
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Kernel Livepatch</h3>
+          <div class="p-matrix__desc">
+            <p>Ubuntu kernel updates are regularly issued. With our Livepatch service, kernel patches are delivered immediately, without the need to reboot your VMs.</p>
+          </div>
+        </div>
+      </li>
+    </ul>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>24/7 supported SQL Server on Ubuntu Pro. Backed by Canonical and Microsoft.</h2>
+    </div>
```

Comment:
It looks a little odd wrapped onto three lines, perhaps we could break it into two lines like this:
```suggestion
    <div class="col-12">
      <h2>24/7 supported SQL Server on Ubuntu Pro.<br>Backed by Canonical and Microsoft.</h2>
    </div>
```